### PR TITLE
Edit existing links

### PR DESCRIPTION
**Pull request by whatisjasongoldstein**
Small change so that if you select an existing link and press the link button, you get the current href. This seemed like more intuitive UI.

Anyway, I thought I'd push it upstream.

### CODE REVIEW

File: src/wysiwyg.coffee [hunk 1]
```diff
@@ -52,8 +52,14 @@ class Wysiwyg
     e.preventDefault()
     return unless @selectTest()
 
+    href = 'http://'
+    parent_element = window.getSelection().focusNode.parentElement
```
Comment (maccman) [inline src/wysiwyg.coffee]: Can you use camelCasing() ?

File: src/wysiwyg.coffee [hunk 2]
```diff
@@ -52,8 +52,14 @@ class Wysiwyg
...
+    
+    if parent_element.href != undefined
```
Comment (maccman) [inline src/wysiwyg.coffee]: Can you do a check for parentNode's existence:

`if parentElement?.href`

File: src/wysiwyg.coffee (hunk 1 above)
Comment (whatisjasongoldstein) [inline src/wysiwyg.coffee]: Sure.

File: src/wysiwyg.coffee (hunk 2 above)
Comment (whatisjasongoldstein) [inline src/wysiwyg.coffee]: Done. Although there's always a parentNode, even if it's the contenteditable div.

**Comment by maccman** [comment]
Sorry to be so picky here, but one last thing. Can you just do a check for `if parentElement?.href` instead of `if parentElement?.href != undefined`. The former will do checking against empty strings - the undefined check isn't necessary.

**Comment by whatisjasongoldstein** [comment]
That seems reasonable. Updated.

**Pull request merged**